testutil/daemon/daemon.go

```diff
@@ -151,6 +151,10 @@ func New(t testing.TB, ops ...Option) *Daemon {
 
 	assert.Check(t, dest != "", "Please set the DOCKER_INTEGRATION_DAEMON_DEST or the DEST environment variable")
 
+	if os.Getenv("DOCKER_ROOTLESS") != "" {
+		t.Skip("github.com/docker/docker/testutil/daemon.Daemon doesn't support DOCKER_ROOTLESS")
+	}
```

Comment:
Curious why you didn't use `skip.If` here as well

Reply:
Because no `testEnv` in `daemon` pkg